packages/e2e-test-utils/src/rest-api.js

```diff
@@ -61,10 +61,29 @@ const setNonce = ( async () => {
 	);
 
 	// Get the initial nonce.
-	const res = await fetch( apiFetch.nonceEndpoint, {
+	const response = await fetch( apiFetch.nonceEndpoint, {
 		headers: { cookie },
 	} );
-	const nonce = await res.text();
+
+	if ( response.ok || response.status === 302 || response.status === 200 ) {
```

Comment:
I think this just needs to be `if ( response.status === 200 ){` as the nonce endpoint success always seems to be 200, it was the login call that gave a 302.

Reply:
Great, thanks. I'll add it in if this patch works at all. 🤣

Reply:
Maybe we should add a retry limit too similar to what you added in https://github.com/WordPress/gutenberg/pull/37205